sway-lib-std/src/hash.sw

```diff
@@ -418,4 +432,22 @@ fn test_hasher_sha256_bool() {
     true.hash(hasher);
     let sha256 = hasher.sha256();
     assert(sha256 == 0x4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a);
-}
\ No newline at end of file
+}
+
+#[test]
+fn test_hasher_sha256_bytes() {
```

Comment:
No keccak256 tests

Reply:
This can be done in another PR alongside the rest of the file.